content/en/docs/setup/keadm.md

```diff
@@ -15,20 +15,20 @@ Please refer [kubernetes-compatibility](https://github.com/kubeedge/kubeedge#kub
 
 ## Limitation
 
-- Currently support of `keadm` is available for Ubuntu and CentOS OS. RaspberryPi supports is in-progress.
+- Currently support of *keadm* is available for Ubuntu and CentOS OS. RaspberryPi supports is in-progress.
```

Comment:
keadm is also a command. Ideally avoid using bold for commands. This change is not required